ansible/configs/satellite-vm/default_vars_ec2.yml

```diff
@@ -1,17 +1,14 @@
 ################################################################################
-################################################################################
-### Environment Structure
-################################################################################
+### Environment Settings for aws
 ################################################################################
 ## Environment Sizing
 
+cloud_provider: ec2                     # Which AgnosticD Cloud Provider to use                # User info for notifications
+HostedZoneId: Z3IHLWJZOU9SRT
+aws_region: ap-southeast-2
 
-default_key_name: ~/.ssh/{{key_name}}.pem
-tower_run: false
 
 # How many do you want for each instance type
-
-
 bastion_instance_type: "t2.medium"
```

Comment:
Consider moving to t3 for more future proof if you are already making and testing changes.

Reply:
Changed to t3.medium.